app/views/articles/_comment_tree.html.erb

```diff
@@ -0,0 +1,2 @@
+<% comment, sub_comments = comment_node %>
+<%= nested_comments(tree: { comment => sub_comments }, commentable: @article, is_view_root: true) %>
```

Comment:
I'm not too pleased with how it is right now but this is the most quick and dirty implementation I can think of .

It felt touch to try to unfurl `#nested_comments` here because it's a recursive helper, which to my surprised is the [suggestion implementation](https://github.com/stefankroes/ancestry/wiki/Nested-HTML-from-the-arrange-method#recursively-render-a-rails-partial) by ancestry gem

Reply:
> which to my surprise is the suggestion implementation by ancestry gem

I don't find that too surprising, recursion is fairly common in tree traversals. 

Anyway, I think this is ok for now, we can always revisit this at a later date.